configure.ac

```diff
@@ -121,15 +121,38 @@ AC_ARG_WITH([rpc],
     [with_rpc="auto"]
 )
 AS_IF([test x"$with_rpc" != "xno"], [
-    PKG_CHECK_MODULES([TIRPC],[libtirpc], [
-        AC_DEFINE([HAVE_RPCENT_H], [1], [Have <rpc/rpcent.h>.])
-    ], [
-        AS_IF([test x"$with_rpc" = "xyes"], [
-            AC_MSG_ERROR([RPC support requested but not found])
+    rpc_test="libc libtirpc"
+    AS_CASE(["$with_rpc"],
+	[yes], [],
+	[auto], [rpc_test="$rpc_test no"],
+	[*], [rpc_test="`AS_ECHO(["$with_rpc"]) | tr , ' '`"])
+    AS_FOR([], [rpc_impl], [$rpc_test], [
+        AS_CASE(["$rpc_impl"],
+            [libc], [],
+            [no], [break],
+            [*], [PKG_CHECK_MODULES([RPC], ["$rpc_impl"], [], [continue])])
+        have_rpc=maybe
+        AC_CHECK_HEADERS([rpc/rpc.h],, [have_rpc=no; break])
+        # FIXME do a linking test
```

Comment:
you want to use `dnl` for comments in configure.ac in general

Reply:
fixed